### DISCUSSION TITLE
monitors

### DISCUSSION THREAD

**Discussion by Real38YT** [cc-tweaked/CC-Tweaked#1880] [Q&A]
how do i get the text appear on monitors? i do the monitor command but nothing shows

**Comment by fatboychummy** [comment]
What is the exact command that you tried? Does any error message appear on the terminal?

The `monitor` command is meant to run a *program* on a monitor -- it doesn't just write text (unless that's all the program does). Usage is `monitor <monitor name> <program name> [program arguments]` So, for example, if you wanted to run `adventure` on the monitor to your computer's right, you would do `monitor right adventure`.

Is this similar to what you were trying?

**Reply by Real38YT** [reply]
i did monitor top hello and nothing appears on the monitor on the top

**Reply by SquidDev** [reply]
Just to check:

 - What version of Minecraft are you on (and what mod loader)? There's a known bug in 1.21 NeoForge where monitors won't update (fixed in 54a95e07a49378c8dbc8226b2cdf6d6cfdcb82b6, but not released yet).
 - Are you running any shader mods? Some versions of Oculus have a bug which stops monitors rendering.

**Reply by Real38YT** [reply]
no shaders but i am using neoforge 21.0.37 1.21

**Reply by SquidDev** [reply]
Ahh, that'll be it then. I would recommend against using the 1.21 version for now — it's very much alpha quality software, and there will likely be some breaking changes in the near future.
